riotfile.py

```diff
@@ -160,12 +160,15 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
         Venv(
             name="docs",
             pys=["3"],
+            create=True,
             pkgs={
                 "cython": latest,
                 "reno[sphinx]": latest,
                 "sphinx": latest,
                 "sphinxcontrib-spelling": latest,
                 "PyEnchant": latest,
+                "six": latest,
```

Comment:
No idea why but the ddtrace-required dependencies somehow aren't available in the environment that sphinx-build runs

https://app.circleci.com/pipelines/github/DataDog/dd-trace-py/12276/workflows/86ee5d0a-23f9-4591-8b00-7835d22ccf62/jobs/901625

Reply:
probably something important to debug from riot 😟